### DISCUSSION TITLE
How would you integrate tailwindcss with a styled components library?

### DISCUSSION THREAD

**Discussion by zkSoju** [tailwindlabs/tailwindcss#10340] [General]
I'm trying to use https://react95.io/ which is a styled components library but also use tailwind. This is causing some issues with NextJs, is there a proper way to set this up?

**Comment by wongjn** [comment]
For Next.js, you should be able to follow [the installation guide](https://tailwindcss.com/docs/guides/nextjs).

**Reply by zkSoju** [reply]
Issue is that it is overriding some of the styling from the UI library that relies on styled components.
